background/storage.js

```diff
@@ -305,6 +357,10 @@ function saveStyle(style) {
         style, codeIsUpdated, reason,
       });
     }
+    if (!existed && reason === 'install') {
```

Comment:
it'd be more reliable to also check `style.usercss` because in the future we may use reason=install for something else.